operator/cmd/mesh/manifest-apply.go

```diff
@@ -66,9 +66,9 @@ func addManifestApplyFlags(cmd *cobra.Command, args *manifestApplyArgs) {
 	cmd.PersistentFlags().StringSliceVarP(&args.inFilenames, "filename", "f", nil, filenameFlagHelpStr)
 	cmd.PersistentFlags().StringVarP(&args.kubeConfigPath, "kubeconfig", "c", "", "Path to kube config.")
 	cmd.PersistentFlags().StringVar(&args.context, "context", "", "The name of the kubeconfig context to use.")
-	cmd.PersistentFlags().BoolVarP(&args.wait, "wait", "w", false,
+	cmd.PersistentFlags().BoolVarP(&args.wait, "wait", "w", true,
```

Comment:
From Martin: `right. that will be the default going forward. there's little user impact because we've always waited for base and istiod components to be ready`